PreviewThumbnailExtractor/Program.cs

```diff
@@ -107,7 +107,11 @@ private static async Task ProcessThumbnailsAsync(string destination, Cancellatio
                     using (var image = new Image<SixLabors.ImageSharp.PixelFormats.Bgra32>((int)(Pitch / BytePerPixel), (int)Lines))
                     using (var sourceStream = file.file.CreateViewStream())
                     {
-                        sourceStream.Read(MemoryMarshal.AsBytes(image.GetPixelMemoryGroup().Single().Span));
+                        IMemoryGroup<SixLabors.ImageSharp.PixelFormats.Bgra32> mg = image.GetPixelMemoryGroup();
```

Comment:
You didn't set the using on top, so the build is failing. You could also use `var`